tecken/tests/test_sentry.py

```diff
@@ -226,3 +232,33 @@ def test_count_sentry_scrub_error(metricsmock):
     metricsmock.assert_incr(
         "tecken.sentry_scrub_error", value=1, tags=["host:testnode"]
     )
+
+
+def test_sentry_wrap_non_app_error_has_release():
+    port = os.environ.get("EXPOSE_SENTRY_PORT", 8090)
+
+    # Flush fakesentry to ensure we fetch only the desired error downstream
+    requests.post(f"http://fakesentry:{port}/api/flush/")
+
+    current_dir = os.path.dirname(os.path.abspath(__file__))
+    base_dir = os.path.join(os.path.dirname(current_dir), "..")
+    release = get_release_name(base_dir)
+
+    expected_event = {"release": release}
```

Comment:
I couldn't figure out how to mock `get_release_name` when `sentry_wrap.py` is executed in a subprocess. Previously, when I was running this test in the test process, the `unittest` `@patch` decorator did this.

Reply:
Thanks! Yeah I thought about trying to mess with `version.json`, but it seemed more complicated, given that whether or not it exists and what its contents are vary by environment. I'll keep it this way in that case.